packages/api/endpoints/rules.js

```diff
@@ -67,21 +67,21 @@ async function post(req, res) {
   } = req.testContext || {};
 
   let record;
-  const rule = req.body || {};
-  const name = rule.name;
+  const apiRule = req.body || {};
+  const name = apiRule.name;
 
   if (await model.exists(name)) {
     return res.boom.conflict(`A record already exists for ${name}`);
   }
 
   try {
-    rule.createdAt = Date.now();
-    rule.updatedAt = Date.now();
-    const postgresRecord = await translateApiRuleToPostgresRule(rule, dbClient);
+    apiRule.createdAt = Date.now();
+    apiRule.updatedAt = Date.now();
+    const postgresRule = await translateApiRuleToPostgresRule(apiRule, dbClient);
 
     await dbClient.transaction(async (trx) => {
-      await trx(tableNames.rules).insert(postgresRecord, 'cumulus_id');
-      record = await model.create(rule, rule.createdAt);
+      await trx(tableNames.rules).insert(postgresRule, 'cumulus_id');
```

Comment:
Can we create/use the rules PG `create` method here?

Reply:
Yes,  that's already feedback left above and on the list for approval.

Reply:
d2ee549eb

Reply:
d2ee549eb